### PR TITLE
[WIP] Integrated cork into CocoaPods

### PR DESCRIPTION
**Pull request by Karla-Isabel-Sandoval**
Fixes https://github.com/CocoaPods/CocoaPods/issues/3398

### CODE REVIEW

File: spec/cocoapods-integration-specs [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 3c74dd99cf532ba61336e940f79fb8b02467a65b
```
Comment (segiddins) [inline spec/cocoapods-integration-specs]: why this change?

Comment (Karla-Isabel-Sandoval) [inline spec/cocoapods-integration-specs]: To fulfill # 4 on https://github.com/CocoaPods/Cork/issues/3

Comment (segiddins) [inline spec/cocoapods-integration-specs]: but what changed in the integration specs, since there are as of yet no functional changes in the CocoaPods codebase?

**Comment by orta** [comment]
@Karla-Isabel-Sandoval now that your Cork work is merged, are you going to look at continuing this PR?

**Comment by segiddins** [comment]
Closing for the time being -- @Karla-Isabel-Sandoval feel free to re-open when you want to get some review!
